src/core/config.js

```diff
@@ -31,7 +31,10 @@ const configSchema = s({
     enabled: 'boolean?',
     addresses: optional(s(['multiaddr'])),
     interval: 'number?'
-  }, { enabled: true, interval: 30 * 1000 }),
+  }, { // defaults
+    enabled: process.env.NODE_ENV !== 'test', // preload by default, unless in test env
```

Comment:
Process is not available in the browser. All users bundling IPFS will have to configure their bundler to shim a global process object. Can we postpone these changes to a separate PR?

Reply:
Can't postpone this without disabling preload tests.
If we enable preload by default in tests, then p-queue will get filled with requests for ephemeral CIDs that take time to timeout, and actual preload tests will fail.
 
Would it be ok to do:
```js
typeof process === 'undefined' || process && process.env && process.env.NODE_ENV !== 'test'
```
instead?

Reply:
Would the check proposed in https://github.com/ipfs/js-ipfs-utils/pull/10 be acceptable?

Reply:
@alanshaw switched to ipfs-utils v0.2.0 in https://github.com/ipfs/js-ipfs/pull/2304/commits/ad65329253da333885b86c7927aa8f0a1e628551 should be ok to merge now